### DISCUSSION TITLE
plot.plot_response_curves not working

### DISCUSSION THREAD

**Discussion by TomasPapantos** [google/lightweight_mmm#290] [Q&A]
Hi,

I'm running the plot.plot_response_curves but I get this error:
"ValueError: The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()".
Any ideas how to solve it?

Thanks!
Tom

**Comment by iceberg12** [comment]
If the log also tells you the function causing the error is `_compare_equality_for_lmmm`, one of your MMM model attributes is not a proper list of strings (expected) but something else, and you need to fix it.

For example, in my case, when fitting the models, I accidentally passed `media_names` as a pandas series index 
`Index(['Facebook', 'Online Video', 'Paid Search'], dtype='object', name='channel')` 
instead of the correct format 
`['Facebook', 'Online Video', 'Paid Search']`
and it complained.
